### PR TITLE
Sql alchemy stops

### PR DESCRIPTION
**Pull request by Maryl12342**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -372,6 +372,7 @@ services:
 
 volumes:
   postgres-db-volume:
+  postgres-db-bitnami-volume:
```
Comment (klepper24) [inline docker-compose.yml:375]: Is it needed to have 2 volumes? If not, please remove one of them.
In general, I'm not sure if lines 373-375 are needed at all.

**Comment by Maryl12342** [comment]
@klepper24 , @meg25ellele please review my changes.

File: orchestrator/dags/mongo_to_postgres_stops.py [hunk 2]
```diff
@@ -1,24 +1,18 @@
 import os
```
Comment (klepper24) [inline orchestrator/dags/mongo_to_postgres_stops.py]: I think it's no longer needed since you import variables from settings

**Review by klepper24 — CHANGES_REQUESTED** [review]
Because mongo_to_postgres.py it's not a DAG yet, I recommend creating a new folder for such scripts which will be implemented as a DAG in future.

**Comment by Maryl12342** [comment]
Changes added.

**Review by klepper24 — APPROVED** [review]

**Pull request merged**